base/applications/regedit/framewnd.c

```diff
@@ -164,10 +164,36 @@ void SetupStatusBar(HWND hWnd, BOOL bResize)
 
 void UpdateStatusBar(void)
 {
-    HKEY hKeyRoot;
-    LPCWSTR pszKeyPath = GetItemPath(g_pChildWnd->hTreeWnd, 0, &hKeyRoot);
+    HKEY hKeyRoot = NULL;
+    LPCWSTR pszKeyPath = NULL;
+    LPWSTR pszFullPath = NULL;
+    DWORD dwCbFullPath;
+
+    pszKeyPath = GetItemPath(g_pChildWnd->hTreeWnd, 0, &hKeyRoot);
```

Comment:
```suggestion
    pszKeyPath = GetItemPath(g_pChildWnd->hTreeWnd, 0, &hKeyRoot);
    if (!pszKeyPath)
        return;
```
Yes, GetItemPath _can_ return NULL, so if it fails, return early instead of using an invalid value below.